src/MagnumPlugins/TinyGltfImporter/TinyGltfImporter.cpp

```diff
@@ -473,6 +494,12 @@ UnsignedInt TinyGltfImporter::doTextureCount() const {
 
 Containers::Optional<TextureData> TinyGltfImporter::doTexture(const UnsignedInt id) {
     const tinygltf::Texture& tex = _d->model.textures[id];
+
+    if(tex.sampler < 0) {
+        /* The specification instructs to use "auto sampling", i.e. it is left to the implementor to decide on the default values... */
+        return TextureData{TextureData::Type::Texture2D, SamplerFilter::Linear, SamplerFilter::Linear,
+            Sampler::Mipmap::Linear, {Sampler::Wrapping::Repeat, Sampler::Wrapping::Repeat, Sampler::Wrapping::Repeat}, UnsignedInt(tex.source), &tex};
+    }
```

Comment:
Would be great if you could mention the defaults in case no sampler definition is present in the docs.